ENG-8369: Fix policy docs to inform how to use 'any' for number of rows (#201)
cyralinc/terraform-provider-cyral: docs/resources/policy_rule.md

- `deletes` - (Optional) A contexted rule for accesses of the type `delete`.
- `hosts` - (Optional) Hosts specification that limits access to only those users connecting from a certain network location.
- `rate_limit` - (Optional) Rate Limit specifies the limit of calls that a user can make within a given time period.

> - `rows` - (Optional) Rows specification how many rows can be return to the policy rules, we can use integer numbers to define rows, but if you expect return `any` number of rows is only necessary set rows as `-1`.

Reviewer: Both `rate_limit` and `rows` should be moved to a proper section as these are attributes that are part of inner blocks, not the root of the resource. Please see how this is done [here](https://github.com/cyralinc/terraform-provider-cyral/blob/main/docs/resources/repository_local_account.md). Also, also please add the description to the `Description` in the schema definition itself. See [this reference](https://github.com/cyralinc/terraform-provider-cyral/blob/main/cyral/resource_cyral_policy_rule.go#L101).

Author: Perfect, I will update with this changes

For more information, see the [Policy Guide](https://cyral.com/docs/policy#the-rules-block-of-a-policy).

### Reads

- `rate_limit` - (Optional) Rate Limit specifies the limit of calls that a user can make within a given time period.

> - `rows` - (Required) Rows specification how many rows can be return to the policy rules, we can use integer numbers to define rows, but if you expect return `any` number of rows is only necessary set rows as `-1`.

Reviewer: ```suggestion
- `rows` - (Required) How many rows can be return by the policy rule. Use positive integer numbers to define how many rows. If you want to define `any` number of rows, set as `-1`.
```

### Updates

- `rate_limit` - (Optional) Rate Limit specifies the limit of calls that a user can make within a given time period.

> - `rows` - (Required) Rows specification how many rows can be return to the policy rules, we can use integer numbers to define rows, but if you expect return `any` number of rows is only necessary set rows as `-1`.

Reviewer: ```suggestion
- `rows` - (Required) How many rows can be return by the policy rule. Use positive integer numbers to define how many rows. If you want to define `any` number of rows, set as `-1`.
```

### Deletes

- `rate_limit` - (Optional) Rate Limit specifies the limit of calls that a user can make within a given time period.

> - `rows` - (Required) Rows specification how many rows can be return to the policy rules, we can use integer numbers to define rows, but if you expect return `any` number of rows is only necessary set rows as `-1`.

Reviewer: ```suggestion
- `rows` - (Required) How many rows can be return by the policy rule. Use positive integer numbers to define how many rows. If you want to define `any` number of rows, set as `-1`.
```
